connection.go

```diff
@@ -110,17 +110,16 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
 // msgpack array, cause Tarantool always returns result as an array.
 // For all space related methods and Call* (but not Call17*) methods Tarantool
 // always returns array of array (array of tuples for space related methods).
-// For Eval* and Call17* tarantool always returns array, but does not forces
+// For Eval* and Call17* Tarantool always returns array, but does not forces
 // array of arrays.
-
 type Connection struct {
```

Comment:
http://127.0.0.1:6060/pkg/github.com/tarantool/go-tarantool/#ClientError

ClientError is connection produced by this client

I think some words are missing, like `error`